src/nccl_ofi_net.c

```diff
@@ -943,19 +925,28 @@ static ncclResult_t ofi_connect(int dev, void *handle, void **sendComm)
 
 	req = allocate_nccl_ofi_request(sComm->nccl_ofi_reqs_fl);
 	if (OFI_UNLIKELY(req == NULL)) {
-			ret = ncclSystemError;
-			NCCL_OFI_WARN("Unable to get NCCL OFI request for device %d",
-				      sComm->dev);
-			goto error;
+		ret = ncclSystemError;
+		NCCL_OFI_WARN("Unable to get NCCL OFI request for device %d",
+			      sComm->dev);
+		goto error;
+	}
+
+	sComm->local_ep_name = calloc(1, MAX_EP_ADDR);
```

Comment:
We have already inserted remote_ep_addr in this EP's address vector. Why do we need to save it separately in sComm that too as local_ep_name?